components/event/EventCard.js

```diff
@@ -35,6 +35,28 @@ export default function EventCard({ event, username }) {
             )}
         </div>
         <div className="flex-1 space-y-1">
+          {!username && event.userStatus && (
```

Comment:
It should be `username && event.userStatus` as `userName` will always be there

or we can even remove userName and keep it as just `event.userStatus`

Reply:
We want to show the status only on the personal event card, not on the community event card. That's why this condition is there.

Reply:
i think its good to have it on both places but not sure @eddiejaoude any suggestions

Reply:
On the community events card, we are showing all the attendees/speakers on a single card, so this will not work then. Look at this issue #2600

Reply:
Oh if that's the case maybe we can just pass a flag here? along with userName as in the props both `userName` and `name` are present which are technically the same but coming from diff pages

`<EventCard username={data.name} event={event} key={index} isUserPage={true} />`

and in the props

`export default function EventCard({ event, username, isUserPage })`

Reply:
ok